packages/core/config/webpack.common.js

```diff
@@ -250,6 +250,7 @@ module.exports = (webpackEnv) => {
             // .module.css (3), .css (4), .module.scss (5), .scss (6)
             ...getStyleLoaders(isEnvProduction),
             isEnvProduction && {
+              // Will this still work now that it's a .tsx file intead of .js?
               test: path.join(
                 redwoodPaths.base,
                 'node_modules/@redwoodjs/router/dist/splash-page'
```

Comment:
Does anyone know if this matches .tsx files as well? (it used to be splash-page.js, now it's splash-page.tsx)

Reply:
I'm 100% sure this would work.